src/lwc/geTemplates/geTemplates.js

```diff
@@ -184,26 +185,38 @@ export default class GeTemplates extends NavigationMixin(LightningElement) {
                 break;
             case 'delete':
                 this.geListViewComponent.setProperty(IS_LOADING, true);
-
-                deleteFormTemplates({ ids: [row.Id] })
-                    .then(formTemplateNames => {
-                        this.geListViewComponent.setProperty(IS_LOADING, false);
-                        this.geListViewComponent.refresh();
-
-                        const toastMessage = GeLabelService.format(
-                            this.CUSTOM_LABELS.geToastTemplateDeleteSuccess,
-                            formTemplateNames);
-
-                        showToast(toastMessage, '', SUCCESS);
-                    })
-                    .catch(error => {
-                        handleError(error);
-                    });
+                this.handleTemplateDeletion(row.Id);
                 break;
             default:
         }
     }
 
+    /*******************************************************************************************************
+     * @description Handles Form Template Deletion. Currently prevents deletion if Form Templates used by a
+     * Data Import Batch
+     * @param {Id} recordId : Record id of the Form_Template__c to be deleted
+     */
+    handleTemplateDeletion (recordId){
+        deleteFormTemplates({ ids: [recordId] })
+            .then(formTemplateNames => {
+                this.geListViewComponent.setProperty(IS_LOADING, false);
+                this.geListViewComponent.refresh();
+
+                if (formTemplateNames) {
```

Comment:
This check of `formTemplateNames` can be removed also, since `deleteFormTemplates` now always returns an Array or throws an exception.